src/devices/index.js

```diff
@@ -0,0 +1,63 @@
+const match = require('../match');
+
+const { PumpSimulator } = require('./pumpSimulator');
```

Comment:
[nit] Any reason why you aren't just exporting the class directly rather than in an object?

Currently:
```
module.exports = {
    PumpSimulator
};
````
and
```
const { PumpSimulator } = require('./pumpSimulator');
```

versus:
```
module.exports = PumpSimulator;
````
and
```
const PumpSimulator = require('./pumpSimulator');
```

Reply:
Are there some files that export multiple?

Reply:
yeah I will clean those up. Its just a hangover from when I had multiple things in a file